### ISSUE TITLE
Bug in extended_div

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#290]
```jl
julia> extended_div(2..10, -2..5)
(Interval(-Inf, -1.0), Interval(0.4, Inf))
```
The second interval should contain the real number `0.4`, but does not, since
```
julia> big"0.4"
0.4000000000000000000000000000000000000000000000000000000000000000000000000000009
```

cc @yashrajgupta

**Comment by yashrajgupta** [comment]
@dpsanders  Is proper rounding is what needed here?

**Comment by lbenet** [comment]
That function is coded [here](https://github.com/JuliaIntervals/IntervalArithmetic.jl/blob/master/src/intervals/arithmetic.jl#L216); as far as I can see, rounding was not considered!

**Comment by yashrajgupta** [comment]
Yeah, so maybe rounding or type conversion can help. I think we should output Float64 interval for all inputs. This can solve the issue of the big interval as output.

**Comment by lbenet** [comment]
> Yeah, so maybe rounding or type conversion can help

I think you should check how `/` is implemented, to do it as it is done there with respect to rounding.

Regarding conversion, I would avoid it if possible, since it is slow. A faster possibility is `atomic`.

> I think we should output Float64 interval for all inputs. This can solve the issue of the big interval as output.

I disagree. If the input is an `Interval{T}` the output should be a `Tuple` of `Interval{T}`s !

**Comment by dpsanders** [comment]
I believe there is a simple implementation of `x / y` by just splitting up `y` if necessary and then dividing!

**Comment by dpsanders** [comment]
Although that is true, I think it will be faster to just add correct rounding here.

**Comment by dpsanders** [comment]
Fixed by #293.
